### PR TITLE
Fixed Readme.MD

### PR DESCRIPTION
**Pull request by ArisGuimera**
Sample Project Location from Configuring the *Purchases* SDK was wrong

### CODE REVIEW

File: examples/MagicWeather/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ See minimum platform version requirements for RevenueCat's *Purchases* SDK [here
 
 | Feature                          | Sample Project Location                   |
 | -------------------------------- | ----------------------------------------- |
-| 🕹 Configuring the *Purchases* SDK  | [MainActivity.kt](app/src/main/java/com/revenuecat/sample/MainActivity.kt) |
+| 🕹 Configuring the *Purchases* SDK  | [MainActivity.kt](app/src/main/java/com/revenuecat/sample/MainApplication.kt) |
```
Comment (beylmk) [inline examples/MagicWeather/README.md:21]: good catch! do you mind updating the link text as well? :)

Comment (ArisGuimera) [reply examples/MagicWeather/README.md]: Done!

**Review by beylmk — APPROVED** [review]
thanks for submitting this!

**Comment by ArisGuimera** [comment]
You're welcome!

**Review by beylmk — APPROVED** [review]

**Pull request merged**